packages/common/src/index.ts

```diff
@@ -24,6 +25,7 @@ export type { Timestamp, Duration, StringValue } from './time';
 export * from './workflow-handle';
 export * from './workflow-options';
 export * from './versioning-intent';
+export * from './typed-search-attributes';
```

Comment:
Again, be specific. There are things here that are meant to be public APIs, but not `*`.

Reply:
Yup, lack of oversight here. I've made the exports more explicit now.